tmt/cli.py

```diff
@@ -622,7 +623,7 @@ def tests_import(
         types: List[str],
         nitrate: bool,
         polarion: bool,
-        polarion_case_id: Optional[str],
+        polarion_case_id: Optional[List[str]],
```

Comment:
I think this can no longer be `Optional` - with `multiple=True`, even the default would be an empty iterable, never `None`. Therefore you can narrow the type a bit. Seems to apply to functions down the stream as well.